libbeat/template/processor.go

```diff
@@ -208,6 +209,21 @@ func (p *Processor) object(f *common.Field) common.MapStr {
 	}
 
 	switch f.ObjectType {
+	case "scaled_float":
```

Comment:
Probably we will have a request for float and integer here pretty soon. As soon this pops up we should refactor the code to remove the duplicated parts, if possible

Reply:
Just saw further below that long we already have. float should be rather simple to add in the same manner.

Reply:
For scaled float I could reuse the existing `scaledFloat` method.